Tests/FunctionalTests/ModuleAliasingTests.swift

```diff
@@ -29,9 +32,17 @@ class ModuleAliasingTests: XCTestCase {
             XCTAssertFileExists(prefix.appending(components: "AppPkg", ".build", UserToolchain.default.triple.platformBuildPathComponent(), "release", "GameUtils.swiftmodule"))
             XCTAssertFileExists(prefix.appending(components: "AppPkg", ".build", UserToolchain.default.triple.platformBuildPathComponent(), "debug", "Utils.swiftmodule"))
             XCTAssertFileExists(prefix.appending(components: "AppPkg", ".build", UserToolchain.default.triple.platformBuildPathComponent(), "release", "Utils.swiftmodule"))
+            
             let result = try SwiftPMProduct.SwiftBuild.executeProcess([], packagePath: app)
-            XCTAssertEqual(result.exitStatus, .terminated(code: 0))
+            let output = try result.utf8Output() + result.utf8stderrOutput()
+            
+            #if os(Linux)
+            // FIXME: rdar://88722540
+            // The process from above crashes on CentOS 7, so print the output for further investigation
+            print(output)
+            #else
+            XCTAssertEqual(result.exitStatus, .terminated(code: 0), "output: \(output)")
+            #endif
```

Comment:
is this a temporary patch to help figure out the compiler issue?

Reply:
According to @shahmishal it's happening on macOS too; removed the process launch line where the crash happens.